src/libs/protocols/protos/agent.proto

```diff
@@ -505,3 +507,55 @@ message GetMetricsRequest {}
 message Metrics {
 	string metrics = 1;
 }
+
+message VolumeStatsRequest {
+	string path = 1;
+}
+
+message VolumeStatsResponse {
+	// This field is OPTIONAL.
+	repeated VolumeUsage usage = 1;
```

Comment:
why is it `repeated`?

Reply:
This is how it's defined in the [CSI spec ](https://github.com/container-storage-interface/spec/blob/master/csi.proto#L1439). `VolumeUsage` has two Units: `BYTES` and `INODES`.